tools/process_smc.py

```diff
@@ -117,14 +128,26 @@ def main(args):
                     overwrite=True)
         if smpl_data is not None:
             selected_kinect = 1
-            body_model_cfg = dict(
-                type='SMPL',
-                gender='neutral',
-                num_betas=10,
-                keypoint_src='smpl_45',
-                keypoint_dst='smpl',
-                model_path='xrmocap_data/body_models',
-                batch_size=1)
+            if smpl_type == 'smplx':
+                body_model_cfg = dict(
+                    type='SMPLX',
```

Comment:
```suggestion
                    type=smpl_type,
```

Reply:
Fixed in [2e69819](https://github.com/openxrlab/xrmocap/pull/71/commits/2e6981933fd4c6ca0680021655faa4820900cc15).